### PR TITLE
Add and test the addition of thresholds to compare endpoint

### LINKED ISSUE #10: expose threshold for compare endpoint
Here: 

https://github.com/conbench/conbenchcoms/blob/90b46d795b24aa818038a03fa2c8740eae2ede26/R/compare.R#L10-L25

we should the ability to change these:

https://github.com/conbench/conbench/blob/fb80afebb798dff6608f6b31273501152254c442/conbench/api/compare.py#L136-L137

leaving the defaults alone.

### PR DESCRIPTION
**Pull request by boshek**
Closes #10 

@sayantikabanik still not able to request reviews for some reason.

### CODE REVIEW

File: R/compare.R [hunk 1]
```diff
@@ -3,21 +3,38 @@
 #' @param type the type of comparison to make (one of: runs, benchmarks, batches, commits)
 #' @param baseline the baseline sha of the entity to compare
 #' @param contender the contender sha of the entity to compare
+#' @param zscore_threshold the zscore threshold to mark regressions and improvements. 
+#'   Default is defined at the Conbench api level.
```
Comment (sayantikabanik) [inline R/compare.R:7]: Thanks for the comment, super helpful

File: tests/testthat/test-compare.R [hunk 2]
```diff
@@ -11,3 +11,60 @@ with_mock_dir(test_path("logged-in"), {
...
+
+with_mock_dir(test_path("resp-class"), {
+  test_that("zscore and pairwise thresholds return correct values", {
+    z_thres <- 10
+    percent_threshold <- 20
+
+    resp <- compare(
+      type = "runs",
+      "ee1",
+      "f00d",
```
Comment (sayantikabanik) [inline tests/testthat/test-compare.R:24]: I am unable to comprehend this bit, a comment would be great

**Review by sayantikabanik — APPROVED** [review]
Leaving One thought wrt to a comment, looks great
Thanks Sam

File: tests/testthat/test-compare.R (hunk 2 above)
Comment (boshek) [reply tests/testthat/test-compare.R:24]: Ah. These are simply hex words. Have a look at this: https://nedbatchelder.com/text/hexwords.html

**Pull request merged**